cfme/web_ui/__init__.py

```diff
@@ -3994,6 +3994,24 @@ def controller_name():
     return sel.execute_script('return ManageIQ.controller;')
 
 
+def match_location(controller=None, title=None, summary=None):
+    """Does exact match of passed data
+
+        Returns:
+        :py:class:`bool`
+    """
+    result = []
+    if controller:
+        result.append(controller_name() == controller)
+    if title:
+        result.append(browser_title() == title)
+    if summary:
+        result.append(summary_title() == summary or
+                      sel.is_displayed('//h3[text() == "{}"]'.format(summary)))
```

Comment:
For an example of 2), see Container Node Edit Tags and Manage Policies pages, where the summary_title() function does not locate the form title, as its in an h3 tag.